pktriggercord-cli.c

```diff
@@ -113,6 +113,7 @@ static struct option const longopts[] = {
     {"read_firmware_version", no_argument, NULL, 27},
     {"settings_hex", no_argument, NULL, 28},
     {"dump_memory", required_argument, NULL, 29},
+    {"counter", required_argument, NULL, 30},
```

Comment:
I would choose a different name, because this option is related to the frames option. Like `frame_start` or `frame_start_number`.

Reply:
I chose `counter` because that's what the GUI used. I initially had it as `counter_start` but changed it. I think it pertains more to the file name and `frames` is just the total number of pictures to take, so I didn't use `frame_start`.